Add differentiating background colour (purple) for issue

bipplane opened the pull request:
### Summary:

Fixes #480 

#### Type of change:

(Delete where appropriate)

- ✨ New Feature/ Enhancement

### Changes Made:

- I changed the background colour of issues to purple (a colour that I personally picked) and left pr as-is. I also updated the header to reflect the same colour of its pr/issue.

### Screenshots:

![Screenshot_13](https://github.com/user-attachments/assets/ef461381-d1b9-48f2-871a-fd748f94c010)

### Proposed Commit Message:

```
Add distinct background for issue/PRs
```

<details><summary>
<h3>Checklist:</h3>
</summary>

- [x] I have tested my changes thoroughly.
- [x] I have created tests for any new code files created in this PR or provided a link to a issue/PR that addresses this.
- [x] I have added or modified code comments to improve code readability where necessary.
- [x] I have updated the project's documentation as necessary.

</details>

bipplane commented:
this could serve as a starting point for default colour. 

in the future, maybe we could let users choose their own desired colour for both issues and prs.

damithc commented:
> Nits to fix. Perhaps we should see if @damithc is ok with the colour choice. Otherwise LGTM! 👍

I'm OK with the colour. One downside is that we use purple to indicate the merged status. But I can't think of a better choice either.
One alternative is to use light purple for merged PRs, light red for closed PRs, and light green for open PRs (and light grey for draft PRs?).  
The differentiation between issues and PRs is that issues don't have a background colour while PRs do. But not sure if using multiple bg colors will make the UI look too chaotic 🤔

bipplane commented:
hi all, I have addressed the feedback and pushed new commits, this time using a badge display to differentiate issues/PRs instead of background colours. do let me know your thoughts!

damithc commented:
> This is a tough one. It does help in identify PR and issues especially when both are open. Here is an example <img alt="image" width="405" height="123" src="https://private-user-images.githubusercontent.com/121163666/470614082-4867a489-1d38-487c-85fe-9c00c3020589.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTM0MTc2MzUsIm5iZiI6MTc1MzQxNzMzNSwicGF0aCI6Ii8xMjExNjM2NjYvNDcwNjE0MDgyLTQ4NjdhNDg5LTFkMzgtNDg3Yy04NWZlLTljMDBjMzAyMDU4OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyNVQwNDIyMTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wYWJlZDQ4MDFhN2FlNjM5ZmY2NjA1YzljMzU5M2EwMGRmN2Q4YTNiMTkwNjFmZDdiNWY0YjdhYmJiOGM5M2MxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.bEDJtx6OfeCD6eItIXDh6-zENMitusU5jpZFtxq-KM8">
> 
> Let's see if anyone else has any comments before we merge.

hmm... the icon on the other end already differentiates between  issues and PRs, right? 

<img width="435" height="136" alt="image" src="https://github.com/user-attachments/assets/fe5c4133-b5e8-4ee2-bb6f-6af0ec5340ee" />

The background colours was meant to give a visual cue at a different level e.g., it helps the user catch unbalanced issue-PR distribution immediately even if the user is not looking for it(e.g., someone doing a lot of PRs but without corresponding issues or vice versa -- normally, we expect one PR per issue).
So, I'm not keen on merging the current version.
